dashboard/app/controllers/pd/application/teacher_application_controller.rb

```diff
@@ -24,7 +24,8 @@ def new
           requiredFields: TEACHER_APPLICATION_CLASS.camelize_required_fields,
           accountEmail: current_user.email,
           apiEndpoint: '/api/v1/pd/application/teacher',
-          userId: current_user.id
+          userId: current_user.id,
+          schoolId: current_user.school_info&.school&.id
```

Comment:
we should add `schoolId` to the list of props for `Teacher1920Application.jsx`

Reply:
Fixed